insights/specs/default.py

```diff
@@ -185,7 +185,14 @@ def is_ceph_monitor(broker):
     chkconfig = simple_command("/sbin/chkconfig --list")
     chrony_conf = simple_file("/etc/chrony.conf")
     chronyc_sources = simple_command("/usr/bin/chronyc sources")
-    cib_xml = simple_file("/var/lib/pacemaker/cib/cib.xml")
+    cib_xml = first_of(
+        [
+            simple_file("/var/lib/pacemaker/cib/cib.xml"),
+            head(
+                glob_file("sos_commands/pacemaker/crm_report/*/cib.xml")
+            )
+        ]
+    )
```

Comment:
@csams, @bfahr - This usage seems a good approach, would you please take a look?

And I believe it works well with the `foreach_*` items. By using the "head/first_of", it seems we can implement the "command_with_args" in the following way (another way to avoid wrapping the Parser result into a list when there is only one item in the `"pre_command"`):
```python
pre_sub_cmds = [a,b,....]

cmd_a = head(foreach_execute(pre_sub_cmds, "/bin/cmd %s"))
```